apps/ensadmin/src/app/ponder-client-api/provider.tsx

```diff
@@ -16,7 +15,7 @@ export function Provider({ url, children }: ProviderProps) {
 
   return (
     <PonderProvider client={ponderClient}>
-      <QueryClientProvider>{children}</QueryClientProvider>
+      {children}
```

Comment:
There's a global QueryClientProvider set with the global WagmiProvider. No need for more ;)